OIIInspector/OIIInspector_calls.py

```diff
@@ -0,0 +1,41 @@
+import json
+import tempfile
+
+from OIIInspector.OIIIClient import OIIIClient
+from utils import setup_arg_parser
+import sys
+
+GET_INDEX_IMAGE_PACKAGES_LIST_ARGS = {("--address",): {
+    "help": "Address of the index image",
+    "required": True,
+    "type": str,
+}}
+
+
+def setup_oiii_client(args, file_name):
```

Comment:
I assume this stuff will start making sense once some implementation has been done. So let's leave it as-is and continue with the implementation